src/screens/Login.js

```diff
@@ -1,110 +1,102 @@
+              of this step and even the next steps ahead
             </Text>
-          </View>
+          </Block>
 
-          <View
-            style={{
-              flex: 2,
-              justifyContent: 'space-evenly',
-              alignItems: 'center',
-            }}
-          >
-            <View>
+          <Block flex={2} space="evenly" center>
+            <Block>
               <Input
-                style={styles.inputStyles}
-                borderless
-                bgColor="transparent"
                 type="email-address"
-                placeholder="kanyewest@thegreatkanye.com"
-                label="Email"
-                placeholderTextColor="#757575"
+                placeholder="Email"
+                autoCapitalize="none"
+                style={{ width: width * 0.9 }}
+                onChangeText={text => this.handleChange('email', text)}
               />
               <Input
-                style={styles.inputStyles}
-                borderless
-                bgColor="transparent"
-                placeholder="My awesome password"
-                label="Password"
-                help="Your password must have between 6 and 9 characters"
-                bottomHelp
-                placeholderTextColor="#757575"
                 password
                 viewPass
+                placeholder="Password"
+                style={{ width: width * 0.9 }}
+                onChangeText={text => this.handleChange('password', text)}
               />
-              <View style={{ flexDirection: 'row-reverse' }}>
-                <TouchableOpacity>
-                  <Text size={10} color="rgb(209,0,125)">
-                    Forgot your password?
-                  </Text>
-                </TouchableOpacity>
-              </View>
-            </View>
-
-            <View>
-              <Button color="error" round>
+              <Text
+                color={theme.COLORS.ERROR}
+                size={theme.SIZES.FONT * 0.75}
+                onPress={() => Alert.alert('Not implemented')}
+                style={{ alignSelf: 'flex-end', lineHeight: theme.SIZES.FONT * 2 }}
+              >
+                Forgot your password?
+              </Text>
+            </Block>
+            <Block>
+              <Button
+                round
+                color="error"
+                onPress={() => Alert.alert(
+                  'Sign in action',
+                  `Email: ${email}
+Password: ${password}`,
+                )}
+              >
                 Sign in
               </Button>
-              <TouchableOpacity>
-                <Text
-                  size={14}
-                  center
-                  color="rgb(209,0,125)"
-                  style={{ marginTop: 10 }}
-                >
+              <Button color="transparent" shadowless onPress={() => navigation.navigate('Register')}>
+                <Text center color={theme.COLORS.ERROR} size={theme.SIZES.FONT}>
```

Comment:
I'm not sure if this text is the cause of the problem, but under the "Sign in" button and on top of this text, there's a thin white border. This is especially visible because of the button's shadow.

Reply:
Can you add a screenshot to investigate?